lib/app.dart

```diff
@@ -80,6 +80,7 @@ class _InsporationState extends State<Insporation> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      debugShowCheckedModeBanner: false,
```

Comment:
I actually like having this when having both a release and a dev build installed on the same phone. Can we disable it only for taking screenshots or something?